api/ttn/lorawan/v3/notification_service.proto

```diff
@@ -40,7 +40,23 @@ message Notification {
   EntityIdentifiers entity_ids = 3;
 
   // The type of this notification.
-  string notification_type = 4;
+  // TODO: Replace with type NotificationType in v4.
```

Comment:
Please file issue, reference the issue URL here and mark the issue with label `bump/major`